homeworks/Dmytro.Tyshchenko_DmytroTy/homework_6/task2.js

```diff
@@ -0,0 +1,84 @@
+const horsesList = [['Big Racket', 'Thoroughbred'], ['Shareef Dancer', 'Thoroughbred'],
+    ['Green Monkey', 'Thoroughbred'], ['Annihilator', 'Thoroughbred'],
+    ['Seattle Dancer', 'Thoroughbred'], ['Mystic Park', 'Standardbred'],
+    ['Pine Chip', 'Standardbred'], ['Sardar\t', 'Thoroughbred'],
+    ['Poetin\t', 'Brandenburger'], ['Lord Sinclair', 'Bavarian Warmblood']];
```

Comment:
also. what's the point of creating of array of arrays? it could just be `{ name: 'Roach', breed: 'Witcherhorse' }` object or smth like this... it's more logical